frontend/workflows/projectSelector/src/types.tsx

```diff
@@ -37,17 +37,19 @@ interface BackgroundPayload {
 
 export type Action = BackgroundAction | UserAction;
 
-export interface State {
+export interface ProjectsState {
```

Comment:
Open to discussing this name. `ProjectsState` could be a bit confusing since it's so similar to `ProjectState`. Another option was `HierarchyState`

Reply:
SelectorState, GlobalProjectState? Something like that?